redwoods-insurance-salesforce/force-app/main/default/lwc/incidentAudioPlayer/__tests__/incidentAudioPlayer.test.js

```diff
@@ -98,4 +98,30 @@ describe('c-incident-audio-player', () => {
             );
         });
     });
+
+    // NOTE: there's an issue while testing audio / video tags with sa11y.
```

Comment:
@albarivas - MAYBE consider removing this test until it works?

Reply:
Yeah, I considered removing it, but felt that it's more illustrative to say "don't try to test this, as it wont' won't work", than letting the test unimplemented (which could seem that we just forgot to create it).

Reply:
#fair.